README.md

```diff
@@ -84,6 +61,83 @@ pip3 install lightly
 
 We strongly recommend that you install Lightly in a dedicated virtualenv, to avoid conflicting with your system packages.
 
+
+### Lightly in Action
+
+With lighlty you can use latest self-supervised learning methods in a modular
+way using the full power of PyTorch. Experiment with differnt backbones,
```

Comment:
differnt

Reply:
Oh, you missed `lighlty`. Just saw this one now o.0

Reply:
maybe grep the docs for it?
```
grep -r lighlty docs/
```